src/EntityFrameworkCore.Testing.Common/AsyncEnumerable.cs

```diff
@@ -28,7 +28,7 @@ public AsyncEnumerable(IEnumerable<T> enumerable)
             }
         }
 
-        public IAsyncEnumerator<T> GetAsyncEnumerator(CancellationToken cancellationToken = new())
+        public IAsyncEnumerator<T> GetAsyncEnumerator(CancellationToken cancellationToken = new CancellationToken())
```

Comment:
Please revert this change

Reply:
Unfortunately, [this is a C# 9 feature](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-9#fit-and-finish-features) and is not supported when targeting .NET Standard.

The only option would be to multi-target net5.0 and netstandard2.1 since they [seem to be interoperable](https://docs.microsoft.com/en-us/dotnet/standard/net-standard#when-to-target-net50-vs-netstandard) and one could use the net5.0 target only for the language features.

If you're fine with multi-targeting, I'll change the TargetFramework accordingly. Not sure how nice it plays with the nuspec files but it should just be a matter of specifying lib/netstandard2.1, right?

Reply:
My understanding was that language support was a compiler version rather than the runtime version concern. A quick check shows that I can build and package branch 3.x for efcore5 targeting .netstandard2.1, and have that working as expected in a .netcore3.1 app.

RE nuspec, that's pretty much it. There is a nuspec for both the moq and nsub packages.

![image](https://user-images.githubusercontent.com/47836784/112751771-3fd21900-9002-11eb-9e04-860c9b674013.png)

Reply:
You're right, it is a compiler version concern. The default language version for netstandard2.1 is C# 8 however, so it might need to be set to 9 explicitly (or my setup is just weird and one of the many sdks I have installed makes this a false positive error).

I've reverted the changes now and changed the nuspec target, so it should be good to merge now

Reply:
Language can be set in the project file.

![image](https://user-images.githubusercontent.com/47836784/112774295-d1c23c00-906b-11eb-8168-2c15da11eaa2.png)

Rider allows this to be set via project properties, not sure about other IDEs

![image](https://user-images.githubusercontent.com/47836784/112774350-0209da80-906c-11eb-928a-762076d23e08.png)

Reply:
Fixed and set to language version 9 explicitly